File: raspberry-pi-ios-client/RaspberryPiClient/Data/Infrastructure/Ros/Service/Srv/EmptyService.swift
//
//  EmptyService.swift
//  RaspberryPiClient
//
//  Created by Shusuke Ota on 2025/12/29.
//

import Foundation

/// [std_srvs/srv/Empty.srv](https://github.com/ros2/common_interfaces/blob/jazzy/std_srvs/srv/Empty.srv)
struct EmptyService: RosCallServiceArgsProtocol, RosServiceResponseValuesProtocol {}

File: raspberry-pi-android-client/app/src/main/java/com/shusuke/raspberry_pi_client/data/infrastructure/ros/service/EmptyService.kt
package com.shusuke.raspberry_pi_client.data.infrastructure.ros.service

import kotlinx.serialization.Serializable

/**
 * [std_srvs/srv/Empty.srv](https://github.com/ros2/common_interfaces/blob/jazzy/std_srvs/srv/Empty.srv)
 *
 * 引数・レスポンスともに空のサービス型。
 * `/reset` 等で使用。
 */
@Serializable
object EmptyService
